system-int-test/test/cmr/system_int_test/search/collection_sorting_search_test.clj

```diff
@@ -2,10 +2,10 @@
   "Tests searching for collections using basic collection identifiers"
   (:require
    [clj-time.core :as t]
-   [clj-time.format :as f]
    [clojure.string :as str]
    [clojure.test :refer :all]
    [cmr.common-app.services.search.messages :as msg]
+   [cmr.common.date-time-parser :as p]
```

Comment:
Nitpick: maybe name this alias something like `date-time-parser`?

Reply:
We have been alias this namespace as `p` throughout CMR, I think I will keep it this way to be consistent. If we want to use a different alias, we should do it across CMR. It didn't bother to change the `p` alias because `date-time-parser` is too long and the function that we call is pretty verbose already.